src/main/kotlin/tanvd/grazi/ide/language/latex/LatexSupport.kt

```diff
@@ -1,22 +1,40 @@
 package tanvd.grazi.ide.language.latex
 
 import com.intellij.psi.PsiElement
+import com.intellij.psi.util.PsiTreeUtil
+import nl.hannahsten.texifyidea.psi.LatexGroup
 import nl.hannahsten.texifyidea.psi.LatexMathEnvironment
 import nl.hannahsten.texifyidea.psi.LatexNormalText
+import nl.hannahsten.texifyidea.psi.LatexOpenGroup
 import nl.hannahsten.texifyidea.psi.LatexTypes.NORMAL_TEXT_WORD
+import org.jetbrains.kotlin.idea.conversion.copy.end
+import org.jetbrains.kotlin.idea.conversion.copy.start
 import org.jetbrains.kotlin.psi.psiUtil.parents
+import org.rust.lang.core.psi.ext.isAncestorOf
 import tanvd.grazi.grammar.GrammarChecker
 import tanvd.grazi.grammar.Typo
 import tanvd.grazi.ide.language.LanguageSupport
 import tanvd.grazi.utils.filterForTokens
 
 class LatexSupport : LanguageSupport() {
+    private val ignoredCategories = listOf(Typo.Category.CASING, Typo.Category.PUNCTUATION, Typo.Category.GRAMMAR)
+
     override fun isRelevant(element: PsiElement): Boolean {
         return element is LatexNormalText && element.parents.none { it is LatexMathEnvironment }
+                && PsiTreeUtil.findFirstParent(element) { it is LatexOpenGroup }?.let { latexOpenGroup ->
+            PsiTreeUtil.findFirstParent(element) { it is LatexGroup }?.let { latexOpenGroup.isAncestorOf(it) } ?: false
```

Comment:
What should mean this situation, when `LatexOpenGroup` `isAncestorOf` `LatexGroup`? Move it to function and name accordingly :)

Reply:
(see as an example implementation of Markdown with functions like `isParagraph`, `isHeader` and so on

Reply:
Done